### PR TITLE
PTT Default config

### PR DESCRIPTION
**Pull request by devilflippy**
Add Scholar's Society (Skill Extended) Compat
Add 2 new spawn on Customs.
Change Therapist infill from Crossroad to Railroad to Tarkov

### CODE REVIEW

File: user/mods/Trap-PathToTarkov/configs/The_Escape/config.json [hunk 1]
```diff
@@ -95,6 +95,13 @@
       "access_via": [
         "AnastasiaStash"
       ]
+    },
+    {
+      "id": "WelcomeToTarkov_Scholar_stash",
```
Comment (guillaumearm) [inline user/mods/Trap-PathToTarkov/configs/The_Escape/config.json]: nitpick -> 

```suggestion
      "id": "PathToTarkov_Scholar_stash",
```

File: user/mods/Trap-PathToTarkov/configs/The_Escape/config.json [hunk 2]
```diff
@@ -380,6 +387,30 @@
...
+      "// Trader name": "Scholar's Society",
+      "disable_warning": false,
+      "override_description": true,
+      "location_description": {
+        "ch": "In moutain, near the abandoned village in Shoreline",
+        "cz": "In moutain, near the abandoned village in Shoreline",
+        "en": "In moutain, near the abandoned village in Shoreline",
+        "es-mx": "In moutain, near the abandoned village in Shoreline",
+        "es": "In moutain, near the abandoned village in Shoreline",
+        "fr": "Dans les montagnes,proche du village abandonné de Littoral",
```
Comment (guillaumearm) [inline user/mods/Trap-PathToTarkov/configs/The_Escape/config.json]:
```suggestion
        "fr": "Dans les montagnes, proche du village abandonné de Littoral",
```

File: user/mods/Trap-PathToTarkov/configs/The_Escape/player_spawnpoints.json [hunk 3]
```diff
@@ -143,6 +143,14 @@
     "RUAFAdmin": {
       "Position": [667.3405, 1.064447, -58.31626],
       "rotation": 287.2482
+    },
+    "Railroad To Port": {
+    "Position": [-145.3853, 2.62803984, -23.7549515],
+    "Rotation": 1.93337643
```
Comment (guillaumearm) [inline user/mods/Trap-PathToTarkov/configs/The_Escape/player_spawnpoints.json]:
```suggestion
      "Position": [-145.3853, 2.62803984, -23.7549515],
      "Rotation": 1.93337643
```

**Review by guillaumearm — DISMISSED** [review]
I let some minor comments, otherwise it looks ok to me.

**Comment by devilflippy** [comment]
Committed

**Review by guillaumearm — APPROVED** [review]

**Pull request merged**